bbb/services.py

```diff
@@ -285,6 +285,16 @@ def reflectTasks(self):
             # If takenUntil isn't set, this task has either never been claimed
             # or got cancelled.
             if not t.takenUntil:
```

Comment:
I don't have a strong preference here... I'd say in the current implementation you can have separate comments for each case.